examples/appprotect/README.md

```diff
@@ -41,10 +41,11 @@ $ kubectl create -f cafe.yaml
     ```
 4. Create an Ingress Resource:
 
-    Update the `appprotect.f5.com/app-protect-security-log-destination` annotation from `cafe-ingress.yaml` with the ClusterIP of the syslog service. For example, if the IP is `10.101.21.110`:
+    Update the `appprotect.f5.com/app-protect-security-log-destination` annotation from `cafe-ingress.yaml` with the FQDN of the syslog service. 
+    For example, if the service name is `syslog-svc` and it is in the `default` namespace (You can also use the service ClusterIP):
     ```yaml
     . . .
-    appprotect.f5.com/app-protect-security-log-destination: "syslog:server=10.101.21.110:514"
+    appprotect.f5.com/app-protect-security-log-destination: "syslog:server=syslog-svc.default:514"
```

Comment:
Can we add this directly to the Ingress resource so that we don't need the reader to ask to do that? Previously, the IP was something that would be different from env to env, but now we have a "static" DNS name, so we can just use it in the Ingress annotation.
can we also do a similar update to the waf policy example?  https://github.com/nginxinc/kubernetes-ingress/tree/master/examples-of-custom-resources/waf